src/daemon/wireguardutils.h

```diff
@@ -41,23 +41,9 @@ class WireguardUtils : public QObject {
   virtual bool deletePeer(const InterfaceConfig& config) = 0;
   virtual QList<PeerStatus> getPeerStatus() = 0;
 
-  virtual bool updateRoutePrefix(const IPAddress& prefix) {
-    Q_UNUSED(prefix);
-    return true;
-  }
-  virtual bool deleteRoutePrefix(const IPAddress& prefix) {
-    Q_UNUSED(prefix);
-    return true;
-  }
-
-  virtual bool addExclusionRoute(const IPAddress& prefix) {
-    Q_UNUSED(prefix);
-    return true;
-  }
-  virtual bool deleteExclusionRoute(const IPAddress& prefix) {
-    Q_UNUSED(prefix);
-    return true;
-  }
+  virtual bool updateRoutePrefix(const IPAddress& prefix) = 0;
+  virtual bool deleteRoutePrefix(const IPAddress& prefix) = 0;
+  virtual bool excludeLocalNetworks(const QList<IPAddress>& addresses) = 0;
```

Comment:
Why did we need these updates? (Just trying to better understand this PR.)

Reply:
The `addExclusionRoute()` and `deleteExclusionRoute()` methods weren't being used anywhere so it seemed best to remove them in favor of the new `excludeLocalNetworks()` method.

Reply:
Fully missed those other ones subclassed this one. 🤦🏻